src/definitions/interfaceType.ts

```diff
@@ -1,10 +1,32 @@
 import { assertValidName } from 'graphql'
-import { GetGen } from '../typegenTypeHelpers'
+import { GetGen, InterfaceFieldsFor, FieldResolver, ModificationType } from '../typegenTypeHelpers'
 import { AbstractTypes, NexusTypes, NonNullConfig, RootTypingDef, withNexusSymbol } from './_types'
 import { OutputDefinitionBlock, OutputDefinitionBuilder } from './definitionBlocks'
+import { ArgsRecord } from './args'
 
 export type Implemented = GetGen<'interfaceNames'> | NexusInterfaceTypeDef<any>
 
+export interface FieldModification<TypeName extends string, FieldName extends string> {
+  type?: ModificationType<TypeName, FieldName>
+  /**
+   * The description to annotate the GraphQL SDL
+   */
+  description?: string | null
+  /**
+   * The resolve method we should be resolving the field with
+   */
+  resolve?: FieldResolver<TypeName, FieldName>
+  /**
+   * You are allowed to add non-required args when modifying a field
```

Comment:
Ah I think this answers me previous question, only arg additions are allowed?

Come to think of it this makes sense because otherwise it would violate the interface contract in GraphQL :)

Reply:
Yeah, only arg additions. I didn't think it mattered much to change the contract of existing args.

Reply:
I believe graphql-js will give an error in the schema validation. I wasn't too worried about adding logic for it on our end because it's such an edge case.